packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1113,6 +1118,36 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     return enableInteractiveSelection ?? !obscureText;
   }
 
+  /// The color used to paint the prompt rectangle.
+  Color get promptRectColor => _promptRectPaint.color;
```

Comment:
Should note that this property only impacts non-web iOS applications. Likewise for promptRectRange. 

Should also note that setting promptRectColor to null indicates that the promptRect shouldn't be drawn at all. There should be tests that cover this